source/includes/administration/_pricings.md

```diff
@@ -20,458 +20,761 @@ curl "https://cloudmc_endpoint/rest/pricings" \
-    "effectiveDate": "2020-05-06T00:00:00Z"
-  }
 }
 ```
 
-Required | &nbsp;
-------- | -----------
-`organization.id` <br/>*UUID* | The id of the organization the pricing belongs to. 
-`productCatalogs` <br/>*Array[productCatalogs.id]* | An array of product catalogs the pricing can add products from.
-`productCatalogs.id` <br/>*UUID* | The id for a product catalog you'd like to add as a source for your pricing.
-`name` <br/>*Object* | A map of language keys to a name in the specified language. 
-`effectiveDate` <br/>*string* | The date the pricing should become effective.
-`pricingProducts` <br/>*Array[pricingProduct]* | A list of products from a supplied product catalog that are priced, need to supply a product with every supported currency.
-`pricingProduct.product.id` <br/>*UUID* | The id of the product you're applying a price to. Must be contained in one of the specified product catalogs. 
-`pricingProduct.cogs` <br/>*int* | The cost of goods sold, represents the direct cost of producing the specified product. 
-`pricingProduct.unitPrice` <br/>*int* | The unit price is the price at which the product will be sold to the consumer.
-`pricingProduct.currency` <br/>*string* | The currency the price is set in. 
-`supportedCurrencies` <br/>*Array[string]*| An array of supported currencies.
-
-Optional | &nbsp;
-------- | -----------
-`description` <br/>*Object* | A map of language language keys to the description in the specified language.
 
 <!-------------------- UPDATE PRICING -------------------->
-#### Update pricing
+### Update Pricing 
 
 `PUT /pricings/:id`
 
-Update an existing pricing's name and description.
+Update the name and description of a pricing. If you want to modify anything else, then you can use the pricing change API.
 
 ```shell
-# Creates a new pricing
-curl -X PUT "https://cloudmc_endpoint/rest/pricings/03bc22bd-adc4-46b8-988d-afddc24c0cb5" \
-   -H "MC-Api-Key: your_api_key"
+# Update a pricing
+curl -X PUT "https://cloudmc_endpoint/v2/pricings/73eb0d75-03b0-44e2-9cbf-9ad25dff5da5" \
+   -H "MC-Api-Key: your_api_key" \
+   -H "Content-Type: application/json" \
+   -d "request_body"
 ```
-
 > Request body example:
 
-```js
+```json
 {
-  "name": {
-      "en": "name-updated",
-      "fr": "nom-updated",
-      "es": "nombre-updated"
-  },
-  "description": {
-    "en": "pricing-desc-updated",
-    "fr": "desc_fr",
-    "es": "desc_es"
-  }
+    "name": {
+        "en": "Name here",
+        "fr": "Nom ici"
+    },
+    "description": {
+        "en": "Description here",
+        "fr": "Description ici"
+    }
 }
 ```
 
-> The above command return JSON structured like this:
+Required | &nbsp;
+---- | -----------
+`id`<br/>*UUID* | The id of the pricing
+`name`<br/>*Map* | A map from language to name value for that language
+`description`<br/>*Map* | A map from language to description value for that language
+`supportedCurrencies`<br/>*List* | A list of currencies supported by the pricing
```

Comment:
just the name and description is required here

Reply:
oups you're right. done